client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -83,7 +83,7 @@
      * <p>- To reset a reader group to a given StreamCut use
      * {@link ReaderGroupConfig.ReaderGroupConfigBuilder#startFromStreamCuts(Map)}.</p>
      *
-     * All existing readers will have to call {@link ClientFactory#createReader(String, String, Serializer, ReaderConfig)}.
+     * All existing readers will have to call {@link io.pravega.client.EventStreamClientFactory#createReader(String, String, Serializer, ReaderConfig)}.
```

Comment:
Please wrap this line as it exceeds max length.

Reply:
done